pkg/services/rpcsrv/server_test.go

```diff
@@ -1706,65 +1880,79 @@ func TestRPC(t *testing.T) {
 }
 
 func TestSubmitOracle(t *testing.T) {
+	rpc := `{"jsonrpc": "2.0", "id": 1, "method": "submitoracleresponse", "params": %s}`
+
+	t.Run("OracleNotRun", func(t *testing.T) {
+		chain, rpcSrv, httpSrv := initClearServerWithServices(t, false, false, false)
+		defer chain.Close()
+		defer rpcSrv.Shutdown()
+		req := fmt.Sprintf(rpc, "[]")
+		body := doRPCCallOverHTTP(req, httpSrv.URL, t)
+		// TODO: get this error
+		//checkErrGetResult(t, body, true, neorpc.ErrOracleDisabledCode)
+		checkErrGetResult(t, body, true, neorpc.InvalidParamsCode)
+	})
+
```

Comment:
@AnnaShaleva I also wish to get error ErrOracleDisabledCode , but failed at InvalidParamsCode. How to change this test case?

Reply:
Create a new RPC server with custom configuration. Disable Oracles in the configuration (`ApplicationConfiguration`/`Oracle`/`Enabled` -> false). See the `pkg/config` package.